apps/src/aiDifferentiation/AiDiffBotMessageFooter.tsx

```diff
@@ -1,12 +1,23 @@
+import {PDFDownloadLink} from '@react-pdf/renderer';
 import React from 'react';
 
 import Button from '@cdo/apps/componentLibrary/button/Button';
 import {commonI18n} from '@cdo/apps/types/locale';
 
+import AiDiffPdf from './AiDiffPdf';
 import {ChatTextMessage} from './types';
 
 import style from './ai-differentiation.module.scss';
 
+const copyToClipboard = (text: string) => {
```

Comment:
Can you put a comment in here to say this is just a fallback for older browsers that lack `navigator.clipboard` support?

Related: do we even claim to support any such browsers anymore?

Reply:
Done